src/adcm_pytest_plugin/steps/actions.py

```diff
@@ -79,7 +81,12 @@ def _run_action_and_assert_result(
     with allure.step(
         f"Perform action '{action_name}' for {obj.__class__.__name__} '{obj_name}'"
     ), _suggest_action_if_not_exists(obj, action_name):
-        task = obj.action(name=action_name).run(**kwargs)
+        if rpm.compare_versions(obj.adcm_version, "2021.02.04.13") >= 0:
+            if "verbose" in kwargs:
+                verbose = kwargs.pop("verbose")
+            else:
+                verbose = options.verbose_actions
+        task = obj.action(name=action_name).run(verbose=verbose, **kwargs)
```

Comment:
In my opinion, for older versions, this key should not be present in `run()` call

Reply:
This block is about setting custom value. If we have an old version and for some reason user passes the 'verbose' key it will be passed within kwargs

Reply:
Oh. My bad. Will fix it